### DISCUSSION TITLE
Cannot change screen locking background on Debian

### DISCUSSION THREAD

**Discussion by CicadaSeventeen** [BuddiesOfBudgie/budgie-desktop#433] [General]
I am using budgie 10.7.1 on debian.
In theory, budgie will set screen locking wallpaper (which is controlled by `gnome-screensaver`, not to confused with `lightdm` greeter inerface) the same as desktop;s wallpaper. However, it does not work on debian. Whatever I set as wallpaper, the screen locking background of  `gnome-screensaver` is always not changed, always the Colosseum.
I also have try `wallstreet` but it does not work.
I try on Ubuntu Budgie and everything is just fine.

**Comment by CicadaSeventeen** [comment]
@fossfreedom Do you have any idea about it?

**Comment by fossfreedom** [comment]
https://discourse.ubuntubudgie.org/t/how-to-change-the-lock-screen-wallpaper-in-debian-10-with-the-budgie-de/2602

Dunno - a quick poke in Debians gnome-screensaver code and the lock-screen is suppose to take its image from org.gnome.desktop.backgrounds - but manually changing that in dconf-editor doesn't see to change anything.

Needs more playing to figure out what is going on.
